packages/fxa-shared/dto/auth/payments/iap-subscription.ts

```diff
@@ -0,0 +1,74 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+import Stripe from 'stripe';
+import joi from 'typesafe-joi';
+
+import {
+  MozillaSubscriptionTypes,
+  SubscriptionTypes,
+} from '../../../subscriptions/types';
+
+export type IapExtraStripeInfo = {
+  product_id: Stripe.Product['id'];
+  product_name: Stripe.Product['name'];
+  price_id: Stripe.Plan['id'];
```

Comment:
We don't want to bind our types to Stripe types as that will make our code more dependent on the specific version of the Stripe library in use. I believe I removed most of these from other locations in fxa-shared.